CMakeLists.txt

```diff
@@ -242,75 +268,63 @@ message(STATUS "Portability strategy set to ${PORTABILITY_STRATEGY}")
 # we may need to rename this variable.
 if (SINGULARITY_BUILD_CLOSURE)
   if(SINGULARITY_USE_KOKKOSKERNELS)
-    # type: external
-    singularity_import_dependency(
-      PKG KokkosKernels
-      TARGETS Kokkos::kokkoskernels
-      )
+    # kokkoskernels solvers
+    # Disable TPLs
+    list(APPEND _kk_tpls "BLAS" "MKL" "LAPACK" "CUBLAS" "CUPARSE" "MAGMA")
```

Comment:
This is brilliant! 

It won't collide with an externally installed kokkos kernels correct?